region/maridia/outer/Maridia Tube.json

```diff
@@ -3223,7 +3223,7 @@
         {"or": [
           "Gravity",
           "HiJump",
-          "h_maxHeightSpringBallJump",
+          "canTrickySpringBallJump",
```

Comment:
trickyJump is there because the unmorph is unintuitive for hard.  But that isn't so important for GMode strats.